### ISSUE TITLE
Change response status to 201 on adding new elements and 202 on editting new elements

### ISSUE THREAD

**Issue by camilosampedro** [camilosampedro/Aton#51]
1. Change controller HTTP status response on `add` methods from `Ok` (`200`) to `Created` (`201`)
2. Change controller HTTP status response on `edit` methods from `Ok` (`200`) to `Accepted` (`202`)

All controllers can be found [here](https://github.com/camilosampedro/Aton/tree/master/app/controllers).

**Comment by raghavraman** [comment]
Hi, Can I take this up and start working.

**Comment by camilosampedro** [comment]
Of course! If you need to know anything about the project, just tell me.
